sdk/core/core-http/src/policies/exponentialRetryPolicy.ts

```diff
@@ -43,10 +41,6 @@ export function exponentialRetryPolicy(
   };
 }
 
-const DEFAULT_CLIENT_RETRY_INTERVAL = 1000 * 30;
-const DEFAULT_CLIENT_RETRY_COUNT = 3;
-const DEFAULT_CLIENT_MAX_RETRY_INTERVAL = 1000 * 90;
```

Comment:
I didn't change any of the default values. The changes in core-https are great but I am concerned about changing current behaviors.